preparing-to-code/keyboard-navigation/README.md

```diff
@@ -1,55 +1,123 @@
+| Command-Space              |   `⌘ + ␣`   | Turn on spotlight.  Spotlight lets you search applications, files and do basic functions like math & conversions.                                                                         |
+| Command-Shift-leftbracket  | `⌘ + ⇧ + [` | Move one tab to the left.                                                                                                                                                                 |
+| Command-Shift-rightbracket | `⌘ + ⇧ + ]` | Move one tab to the right.                                                                                                                                                                |
+| Command-Shift-t            | `⌘ + ⇧ + t` | open the last closed item.                                                                                                                                                                |
+| Command-f                  |   `⌘ + f`   | Find.  Opens a panel to search for a term.                                                                                                                                                |
+| Command-shift-f            | `⌘ + ⇧ + f` | Find all.  Searches all files (text editors like VS Code & Atom only).                                                                                                                    |
+| Command-n                  |   `⌘ + n`   | Opens a new window (or file depending on the application).                                                                                                                                |
+| Command-t                  |   `⌘ + t`   | Opens a new tab in a browser                                                                                                                                                              |
+| Command-r                  |   `⌘ + r`   | Reloads the page in a browser                                                                                                                                                             |
+| Command-comma              |   `⌘ + ,`   | Opens an app's preferences page.                                                                                                                                                          |
+| Command-/                  |   `⌘ + /`   | Make either the current line or a selection of text a comment (text editors like VS Code & Atom only)                                                                                     |
+| Control-Command-f          | `^ + ⌘ + f` | Fullscreen                                                                                                                                                                                |
```

Comment:
You should do a similar thing here but composing the suggestion in the comment window is a PITA so I will leave that for you to do in an editor.